### ISSUE TITLE
Consider declaring aria-readonly as being in direct conflict with contenteditable

### ISSUE THREAD

**Issue by joanmarie** [w3c/html-aria#33]
According to the [ARIA spec's section 7.5](https://rawgit.com/w3c/aria/master/aria/aria.html#host_general_conflict):

> When a host language declares a WAI-ARIA attribute to be in direct semantic conflict with a native attribute for a given element, user agents MUST ignore the WAI-ARIA attribute and instead use the host language attribute with the same implicit semantic.

[...]

> Therefore, to prevent providing conflicting states and properties to assistive technologies, host languages MUST explicitly declare where the use of WAI-ARIA attributes on each host language element conflicts with native attributes for that element.

I cannot find any declaration that `aria-readonly` is in direct semantic conflict with `contenteditable`. Thus given:

`<p contenteditable aria-readonly="true">Edit me!</p>`

The text quoted above would suggest user agents are not _required_ to ignore `aria-readonly`.

Furthermore, at the top of that same section in the ARIA spec, it says:

> Except for the cases outlined below, user agents MUST always use the WAI-ARIA semantics to define how it exposes the element to accessibility APIs, rather than using the host language semantics.

So unless my paragraph example above is a "case outlined below (in section 7.5)," user agents MUST tell ATs that something which is editable by virtue of the `contenteditable` attribute is readonly.

Taking a look, the only thing in the "cases outlined below" I'm finding that could be used to prevent user agents from being obligated to convey this conflicting/erroneous information is:

> there are elements that are inappropriate to override with WAI-ARIA. This could be because identical host language semantics exist, so WAI-ARIA is not needed, or because semantics from WAI-ARIA directly conflict with host language semantics.

I think the example above falls under the heading of "inappropriate to override ... because semantics from WAI-ARIA directly conflict with host language semantics." But I'm not sure that everyone would agree. See, for instance, this [WebKit Layout Test](https://trac.webkit.org/browser/trunk/LayoutTests/accessibility/aria-readonly.html#L8) which includes these test cases:

```
8   <!-- These reflect the native writable state, but can be overridden by @aria-readonly. -->
[...]
10  <div contenteditable id="htmlEditableDiv2" aria-readonly="true" data-expectedwritable="false"></div>
[...]
13  <div role="group" contenteditable id="htmlEditableDiv5" aria-readonly="true" data-expectedwritable="false"></div>
```

It seems to me that explicitly declaring `aria-readonly` in direct conflict with `contenteditable` would ensure that user agents ignore the value of the former attribute for elements which have the latter.

**Comment by stevefaulkner** [comment]
@Joanmarie suggest you file a bug against the HTML AAM about this as well as this spec only sets author conformance requirements.
